packages/ember-data/lib/system/model/internal-model.js

```diff
@@ -135,9 +135,12 @@ InternalModel.prototype = {
   },
 
   finishedReloading: function() {
-    this.isReloading = false;
-    if (this.record) {
-      set(this.record, 'isReloading', false);
+    var record = this;
+    if (record._loadingPromiseChain._state) {
```

Comment:
_state is not a public promises method, especially as it's underscored. cc @stefanpenner
you might have to do this manually for now